developer/environment-setup/index.md

```diff
@@ -0,0 +1,359 @@
+---
+layout: dev-doc
+title: Development Environment Setup
+id: "environment-setup"
```

Comment:
will move this to howto navigation. But want to keep the id with proper name (just to make consistent with other pages).